core/src/cats/sql.cc

```diff
@@ -566,7 +566,7 @@ int BareosDb::ListResult(void* vctx, int nb_col, char** row)
           value.bsprintf(" %-*s |", max_len, "NULL");
         } else if (SqlFieldIsNumeric(field->type) && !jcr->gui
                    && IsAnInteger(row[i])) {
-          value.bsprintf(" %*s |", max_len, add_commas(row[i], ewc));
+          value.bsprintf(" %*s |", max_len, add_separator(row[i], ewc,','));
```

Comment:
There's a space missing.
We have an automated tool `devtools/bareos-check-sources` for things like that. Take a look at the documentation in `devtools/pip-tools/README.md` on how to get it running.

```suggestion
          value.bsprintf(" %*s |", max_len, add_separator(row[i], ewc,','));
```